instrumentation/spring/spring-cloud-gateway/spring-cloud-gateway-2.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/spring/v2_0/ServerWebExchangeHelper.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.spring.v2_0;
+
+import static org.springframework.cloud.gateway.support.ServerWebExchangeUtils.GATEWAY_ROUTE_ATTR;
+
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.context.Context;
+import org.springframework.cloud.gateway.route.Route;
+import org.springframework.web.server.ServerWebExchange;
+
+public final class ServerWebExchangeHelper {
+
+  /** Route info key. */
+  public static final String ROUTE_INFO_ATTRIBUTES = "ROUTE_INFO";
```

Comment:
usually we use something like `private static final AttributeKey<String> ROUTE_INFO_KEY = AttributeKey.stringKey("spring-cloud-gateway.route_info");`